Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,41 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        alert_data = dict_safe_get(alert, ['alert_data', 'csv'])
+        incident_csv_records = dict_safe_get(alert, ['alert_data', 'csv', 'content']) or []
 
         incident = {
             'name': f'Cyberint alert {alert_id}: {alert_title}',
             'occurred': datetime.strftime(alert_created_time, DATE_FORMAT),
             'rawJSON': json.dumps(alert),
             'severity': SEVERITIES.get(alert.get('severity', 'low'), 1),
-            'attachment': incident_attachments
+            'attachment': incident_attachments,
         }
-        incidents.append(incident)
+
+        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
+        if alert_csv_id:

```

Comment:
Let's use safe_dict_get here, with the default return value as the empty string.  In addition, it seems that a walrus operator can be very helpful here :)